about.html

```diff
@@ -0,0 +1,228 @@
+          encadenados en forma de hilos. La nueva función respondía a la demanda
+          de los miles de usuarios que habían empezado a encadenar tuits en las
+          respuestas.
+        </p>
+        <p class="about-event-paragraph">
+          En el verano del 2017, el guionista e ilustrador
+          <a
+            href="https://twitter.com/ManuelBartual"
+            rel="noreferrer noopener"
+            target="_blank"
+            >@ManuelBartual</a
+          >, publicó una fantástica historia de ficción, en la que contaba el
+          encuentro con un doble mientras veraneaba en sus vacaciones. La
+          historia narrada en hilos rápidamente se viralizó hasta convertirse en
+          trending topic mundial, mantuvo en vilo durante 7 días a miles de
+          usuarios pendientes de cada actualización. “<a
+            href="https://lahiloteca.com/hilo/todo-esta-bien/"
+            rel="noreferrer noopener"
+            target="_blank"
+            >Todo está bien</a
+          >” fue publicada en una serie tuits -no mayores de 180 caracteres- que
+          se iban enlazando para contar la historia. Manuel extendió la
+          literatura al formato de hilos. Pronto se popularizó este género
+          narrativo.
+        </p>
+        <p class="about-event-paragraph">
+          En junio del 2018
+          <a
+            href="https://twitter.com/twitterespana?lang=es"
+            rel="noreferrer noopener"
+            target="_blank"
+            >@TwitterEspana</a
+          >
+          convocó al primer certamen de
+          <a
+            href="https://www.feriadelhilo.com/"
+            rel="noreferrer noopener"
+            target="_blank"
+            >La Feria del Hilo</a
+          >
+          , desde entonces se ha convertido en un clásico dentro de Twitter.
+          <a
+            href="https://twitter.com/feriadelhilo"
+            rel="noreferrer noopener"
+            target="_blank"
+            >@FeriaDelHilo</a
+          >
+          es uno de los eventos más esperados, a la sombra de la
+          <a
+            href="https://www.ferialibromadrid.com/"
+            rel="noreferrer noopener"
+            target="_blank"
+            >Feria del Libro de Madrid</a
+          >. En el 2020, durante el confinamiento por la pandemia,
+          <a
+            href="https://twitter.com/twitterespana?lang=es"
+            rel="noreferrer noopener"
+            target="_blank"
+            >@TwitterEspana</a
+          >
+          lanzó una edición especial del certamen de tuiteratura con el hashtag
+          #yoMeQuedoEnCasa
+        </p>
+        <p class="about-event-paragraph">
+          <em>Expo tuiteratura 2022</em>, sigue esta tendencia, a modo de una
+          aventura literaria abrazada por jóvenes universitario, en tierras
+          regias, mexicanas.<br />
+        </p>
+        <img src="resources/images/logo.png" alt="" />
+      </section>
```

Comment:
- [ ] Great job so far, in the design for the about page the logo has borders, please give the logo a border.

### Behance Design

![image](https://user-images.githubusercontent.com/59416136/163455704-6ae78036-86bd-4314-a72c-def91a308cc4.png)

### Your Implementaion

![image](https://user-images.githubusercontent.com/59416136/163455448-b81af1b2-b027-4adf-af7c-5ffe098ea2e8.png)